lib/beaker/dsl/install_utils/module_utils.rb

```diff
@@ -133,8 +133,12 @@ def copy_module_to(one_or_more_hosts, opts = {})
               #move to the host
               logger.debug "Using scp to transfer #{source_path} to #{target_path}"
               scp_to host, source_path, target_module_dir, {:ignore => ignore_list}
+
               #rename to the selected module name, if not correct
               cur_path = File.join(target_module_dir, source_name)
+              if host.is_powershell? #make sure our slashes are correct
+                cur_path = cur_path.gsub(/\//,'\\')
```

Comment:
I'm concerned that we're conflating multiple things here.  There are 2 things that Beaker should really care about:

* Network transport - i.e. `ssh` vs `winrm`
* Interpreter - i.e. `bash`, `cmd`, `powershell`, etc

The problem is that @cowofevil is running Bitvise SSH, and he assumed we should be setting `is_cygwin: false`.

But that doesn't really convey what we care about, and since Beaker doesn't appear to track network transport  apart from interpreter, we end up in a strange state. `is_cygwin: false` appears to end up setting `is_powershell?` to true, which IMHO is not accurate.

Bitvise templates use `ssh` (like Cygwin), but the interpreter used is `cmd`, not `powershell`.